java/src/main/native/src/row_conversion.cu

```diff
@@ -915,23 +845,23 @@ __global__ void copy_strings_to_rows(size_type const num_rows, size_type const n
   // fill out offset/length information for the column. Then all threads of the warp will
   // participate in the memcpy of the string data.
   auto my_block = cooperative_groups::this_thread_block();
-  auto my_tile = cooperative_groups::tiled_partition<32>(my_block);
+  auto warp = cooperative_groups::tiled_partition<NUM_THREADS_IN_WARP>(my_block);
 #ifdef ASYNC_MEMCPY_SUPPORTED
   cuda::barrier<cuda::thread_scope_block> block_barrier;
 #endif
 
   auto const start_row =
-      blockIdx.x * NUM_STRING_ROWS_PER_BLOCK_TO_ROWS + my_tile.meta_group_rank() + batch_row_offset;
+      blockIdx.x * NUM_STRING_ROWS_PER_BLOCK_TO_ROWS + warp.meta_group_rank() + batch_row_offset;
   auto const end_row =
       std::min(num_rows, static_cast<size_type>(start_row + NUM_STRING_ROWS_PER_BLOCK_TO_ROWS));
 
-  for (int row = start_row; row < end_row; row += my_tile.meta_group_size()) {
+  for (int row = start_row; row < end_row; row += warp.meta_group_size()) {
```

Comment:
I vaguely remember that there were out-of-bound bugs related to using `int` for iterating index in kernels. So please be aware of.

Reply:
Offsets are limited to size_type for a cudf column, which is an int. This should be acceptable until that changes.